appveyor.yml

```diff
@@ -25,12 +25,12 @@ install: |
   nvcc -V
 
 build_script:
-  - cmake -G "Visual Studio 14 2015 Win64" -H. -Bbuild -DETHASHCUDA=ON -DAPICORE=ON -DHUNTER_JOBS_NUMBER=%NUMBER_OF_PROCESSORS%
+  - cmake -G "Visual Studio 15 2017 Win64" -H. -Bbuild -T v140,host=x64 -DETHASHCUDA=ON -DAPICORE=ON -DHUNTER_JOBS_NUMBER=%NUMBER_OF_PROCESSORS%
```

Comment:
Why v140 is needed?

Reply:
Because otherwise the default toolset will be used, which isn't the 2015 one.

Reply:
Why do we want the 2015 one?

Reply:
Because it doesn't work with CUDA. Not sure if this is fixed with 9.2. I will try 9.2 later.